packages/reports/src/sass/app/modules/smart-chart.scss

```diff
@@ -316,6 +316,10 @@
     }
 }
 
+.smartcharts .barrier-area {
+    z-index: 1 !important;
```

Comment:
For all barriers with shadow type not equal to `NONE_DOUBLE`, `z-index` is 1. But for mentioned type, z`-index` is 101 and written inline in SmartCharts. `NONE_DOUBLE` is applied only for `Lookbacks High-Low` (packages/shared/src/utils/constants/barriers.ts) and for contract with `barrier_cont` === 2. At the moment it's also `Lookbacks High-Low` .